packages/table/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "@freshworks/table",
+  "version": "0.1.1",
+  "description": "Table component in Nucleus",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "https://github.com/freshdesk/nucleus",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build",
+    "start": "ember serve -p 4003",
+    "deploy": "ember deploy production",
+    "test": "ember backstop-remote & COVERAGE=TRUE ember test --test-port=1509",
+    "posttest": "ember backstop-stop",
+    "test:dev": "COVERAGE=TRUE ember test --server -launch=false"
+  },
+  "dependencies": {
+    "@freshworks/button": "^0.6.3",
+    "@freshworks/core": "^0.2.1",
+    "@freshworks/icon": "^0.8.0",
+    "@freshworks/pagination": "^0.1.1",
+    "ember-cli-babel": "^7.11.1",
+    "ember-cli-htmlbars": "^4.0.0",
+    "ember-cli-sass": "^10.0.0",
+    "ember-css-transitions": "^0.1.16",
+    "ember-decorators": "^6.1.1",
+    "ember-decorators-polyfill": "shibulijack-fd/ember-decorators-polyfill#master",
```

Comment:
Can you please change it to reference https://github.com/freshdesk/ember-decorators-polyfill

Reply:
Done